examples/Atlas/Atlas.m

```diff
@@ -16,8 +16,11 @@
       if ~isfield(options,'terrain')
         options.terrain = RigidBodyFlatTerrain;
       end
-      if ~isfield(options,'hands')
-        options.hands = 'none';
+      if ~isfield(options,'hand_right')
+        options.hand_right = 'none';
+      end
+      if ~isfield(options,'hand_left')
+        options.hand_left = 'none';
       end
```

Comment:
I was considering that, but the total number of places it's used is pretty
small -- I've already adapted the DRC code for the new scheme, which I
think is a lot clearer anyway ("hands" is pretty vague, after all). It's
easy to drop back in if people want it?

On Fri, Mar 20, 2015 at 7:09 PM, Robin Deits <notifications@github.com>
wrote:

> In examples/Atlas/Atlas.m
> <https://github.com/RobotLocomotion/drake/pull/881#discussion_r26883575>:
>
> > @@ -16,8 +16,11 @@
> >        if ~isfield(options,'terrain')
> >          options.terrain = RigidBodyFlatTerrain;
> >        end
> > -      if ~isfield(options,'hands')
> > -        options.hands = 'none';
> > +      if ~isfield(options,'hand_right')
> > +        options.hand_right = 'none';
> > +      end
> > +      if ~isfield(options,'hand_left')
> > +        options.hand_left = 'none';
> >        end
>
> should we allow the old 'hands' option to work for backwards compatibility?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RobotLocomotion/drake/pull/881/files#r26883575>.
>